src/csharp/Grpc.Core/GrpcEnvironment.cs

```diff
@@ -218,12 +219,31 @@ public static void SetCompletionQueueCount(int completionQueueCount)
         }
 
         /// <summary>
+        /// By default, gRPC's internal event handlers get offloaded to .NET default thread pool thread (<c>inlineHandlers=false</c>).
+        /// Setting <c>inlineHandlers</c> to <c>true</c> will allow scheduling the event handlers directly to
```

Comment:
Minor: the use of `<c>` around `inlineHandlers` seems intentional here, but so you are aware, there's also [`<paramref>`](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/paramref).